src/L1/rollup/L1MessageQueue.sol

```diff
@@ -67,6 +70,10 @@ contract L1MessageQueue is OwnableUpgradeable, IL1MessageQueue {
     /// @inheritdoc IL1MessageQueue
     uint256 public nextUnfinalizedQueueIndex;
 
+    /// @dev Mapping from message queue index to enqueue timestamp.
+    mapping(uint256 => uint256) private enqueueTimestamp;
+
+    // todo change gap slots
```

Comment:
not know why it's a todo here. and if for removing this todo, the `40` uint256 buffer should be changed by some smaller number?

Reply:
yes, memo in case I forget to change the slot gas.